rails/benchmarks/bm_activerecord_create.rb

```diff
@@ -4,10 +4,7 @@
 
 require_relative 'support/benchmark_rails.rb'
 
-ActiveRecord::Base.establish_connection(
-  adapter: 'sqlite3',
-  database: ':memory:'
-)
+ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'])
```

Comment:
Could you try if this works for Rails 3.2+? I was testing it out the other time and was running into problems. Thats why I resorted back to passing `adapter` and `database`.